The platform function for the registration of imx27 pwm is missing.

Signed-off-by: Gwenhael Goavec-Merou <gwenhael.goavec-merou@armadeus.com>
---

Index: linux-3.7.6/arch/arm/mach-imx/devices-imx27.h
===================================================================
--- linux-3.7.6.orig/arch/arm/mach-imx/devices-imx27.h
+++ linux-3.7.6/arch/arm/mach-imx/devices-imx27.h
@@ -86,3 +86,7 @@ extern const struct imx_spi_imx_data imx
 extern const struct imx_pata_imx_data imx27_pata_imx_data;
 #define imx27_add_pata_imx() \
 	imx_add_pata_imx(&imx27_pata_imx_data)
+
+extern struct imx_mxc_pwm_data imx27_mxc_pwm_data;
+#define imx27_add_mxc_pwm()	\
+	imx_add_mxc_pwm(&imx27_mxc_pwm_data)
